### DISCUSSION TITLE
Does GPIO work ?

### DISCUSSION THREAD

**Discussion by Mostafasaad1** [pythops/jetson-image#101] [Q&A] [answered]
Now I'm planning to build a custom system to have at least Ubuntu 20 or 22 on my Jetson Nano 2GB model to install Ros2 Humble on it.My question is, if I update that image, will it support the GPIO?

**Comment by pythops** [comment, marked as answer]
Yes it will. 
I'll keep this discussion open if you encounter any issue with it, just share it here 
edit: closing the discussion
